CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp/example` are upgraded to v1.20.0. (#4320)
 - The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`are upgraded to v1.20.0. (#4320)
 - Updated configuration schema to include `schema_url` for resource definition and `without_type_suffix` and `without_units` for the Prometheus exporter. (#4727)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/ecs` are upgraded to v1.24.0. (#4803)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/lambda`.are upgraded to v1.24.0. (#4803)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/ec2`.are upgraded to v1.24.0. (#4803)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/eks`.are upgraded to v1.24.0. (#4803)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/gcp`.are upgraded to v1.24.0. (#4803)
+- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda/test`.are upgraded to v1.24.0. (#4803)
```

Comment:
You got some extra "`.`"s (i.e. "```/foo/bar`.are upgraded...```")

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4803/commits/35be77a1d911041a3009387c1578330402d25631